docs/supported-technologies.asciidoc

```diff
@@ -37,15 +37,16 @@ These are the frameworks that we officially support:
 
 [options="header"]
 |=======================================================================
-|Framework |Version |Note
-|<<express,Express>> |^4.0.0 |
-|<<hapi,hapi>> |>=9.0.0 <19.0.0 |
-|<<hapi,@hapi/hapi>> |>=17.9.0 <19.0.0 |
-|<<koa,Koa>> via koa-router or @koa/router |>=5.2.0 <9.0.0 |Koa doesn't have a built in router,
-so we can't support Koa directly since we rely on router information for full support.
-We currently support the most popular Koa router called https://github.com/alexmingoia/koa-router[koa-router]
-|<<restify,Restify>> |>=5.2.0 |
-|<<fastify,Fastify>> |>=0.27.0 |
+|Framework |Version
+|<<express,Express>> |^4.0.0
+|<<hapi,hapi>> |>=9.0.0 <19.0.0
+|<<hapi,@hapi/hapi>> |>=17.9.0 <19.0.0
+|<<koa,Koa>> via koa-router or @koa/router |>=5.2.0 <9.0.0
+2+|Koa doesn't have a built in router, so we can't support Koa directly since we rely on
+router information for full support. We currently support the most popular Koa router called
+https://github.com/alexmingoia/koa-router[koa-router].
+|<<restify,Restify>> |>=5.2.0
+|<<fastify,Fastify>> |>=0.27.0
```

Comment:
What does this change look like? It seems to be switching from three columns to two but koa remains with three columns. Does this render properly?

Reply:
Check it out here: http://apm-agent-nodejs_1428.docs-preview.app.elstc.co/guide/en/apm/agent/nodejs/master/supported-technologies.html#compatibility-frameworks

I think it makes the table easier to read. Thoughts?

Reply:
Cool, seems reasonable to me. 👍